packages/payment-integration-api/src/checkout-buttons/checkout-button-initialize-options.ts

```diff
@@ -16,4 +16,6 @@ export default interface CheckoutButtonInitializeOptions extends CheckoutButtonO
      * The ID of a container which the checkout button should be inserted.
      */
     containerId: string;
+    currencyCode?: string;
```

Comment:
Why do we need to add these here?

Reply:
Moved `storefrontApiToken` to `buyNowInitializeOptions`, as was discussed